### PR TITLE
GRAD_MOMENTUM not used by RMSProp in DQN

### PR DESCRIPTION
**Pull request by brett-daley**
I noticed that `GRAD_MOMENTUM` on the following line is never used:

https://github.com/kenjyoung/MinAtar/blob/1918a2f165307ff516640ef418ed2f9f66ad8840/examples/dqn.py#L45

I think this is likely a bug, since the default momentum for Pytorch's RMSProp is 0 and not 0.95 (see [docs](https://pytorch.org/docs/stable/generated/torch.optim.RMSprop.html)).
This PR fixes it by passing the value into the optimizer constructor.

### CODE REVIEW

**Comment by kenjyoung** [comment]
I believe this is actually intentional, although very poorly documented. I'm a bit rusty on the context, but take a look at the original DQN source code here: https://github.com/google-deepmind/dqn/blob/master/dqn/NeuralQLearner.lua. They actually implement something like centred RMSProp without momentum. I believe in the nature version of the DQN paper they have parameters referred to as squared gradient momentum and gradient momentum which are each set to 0.95. Cross referencing this with the source code it seems like these refer to the decay factor on the gradient and the squared gradient which are fixed to be the same in the centred version of the algorithm (see for example the pytorch version here: https://pytorch.org/docs/stable/generated/torch.optim.RMSprop.html).

**Comment by brett-daley** [comment]
Ah, I see what you mean! The "momentum" terms are referring to the first and second moment estimations in the denominator. I didn't check the original optimizer before opening the PR 😅

In light of this, I think we could do any of the following:
- Delete the unused variable to avoid confusion,
- Add a comment explaining why it's unused, or
- Just close this PR.

**Comment by brett-daley** [comment]
@kenjyoung PR updated!

**Review by kenjyoung — APPROVED** [review]
👍

**Comment by brett-daley** [comment]
@kenjyoung I don't have write access, would you be able to squash and merge for me?

**Comment by kenjyoung** [comment]
For sure, I thought I did already haha.
